backend/runner/runner.go

```diff
@@ -107,6 +117,18 @@ func Start(ctx context.Context, config Config) error {
 	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.RegisterRunner, svc.registrationLoop)
 	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.StreamDeploymentLogs, svc.streamLogsLoop)
 
+	instanceCounter, err = meter.Int64UpDownCounter("ftl.sys.runner.instance",
```

Comment:
Let's make all the metric names `{meter}.{counter}`. Rather than hardcode that naming pattern, we can just set `var meterName` in each individual otel file, and then all the counter names can be constructed as: `fmt.Sprintf("%s.requests", meterName)` (subbing `request` for whatever the counter name is)

Reply:
applied in new branch